.github/workflows/pitest.yml

```diff
@@ -7,6 +7,636 @@ on:
   pull_request:
 
 jobs:
+  pitest-main:
```

Comment:
`name: pitest`
Please rename of workflow, as you had

Reply:
May be "Pitest" as other starts with caps too.

Reply:
Ok, let s align with others.